src/main/java/com/shigotosagashi/jobdomainservice/domain/JobEntity.java

```diff
@@ -0,0 +1,54 @@
+package com.shigotosagashi.jobdomainservice.domain;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.time.Instant;
+import java.util.Random;
+import java.util.UUID;
+
+@Entity
+@Data
+@Builder(toBuilder = true)
+@Setter(AccessLevel.PRIVATE)
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class JobEntity {
+
+  @Id
+  String id;
+  String title;
+  String description;
+  long creationDate;
+  long expirationDate;
+
+  public static JobEntity create(String id,
+                                 String title,
+                                 String description,
+                                 long creationDate,
+                                 long expirationDate) {
+    return JobEntity.builder()
+        .id(id)
+        .title(title)
+        .description(description)
+        .creationDate(creationDate)
+        .expirationDate(expirationDate)
+        .build();
+  }
+
+  public static JobEntity mock() {
+    return JobEntity.builder()
+        .id(UUID.randomUUID().toString())
+        .title(UUID.randomUUID().toString())
+        .description(UUID.randomUUID().toString())
+        .creationDate(Instant.now().toEpochMilli())
+        .expirationDate(Instant.now().toEpochMilli() + new Random().nextLong())
```

Comment:
Adding a random number to epoch milli could lead us to weird and hard to test dates. There's no need for this.
If you'd want to validate that the expiration date is after the creation one, you should do that inside the test itself. For example:

```
    Instant currentDate = Instant.now();
    Instant expirationDate = currentDate.plus(new Random().nextInt(100) + 1, ChronoUnit.MINUTES);

    assertThat(expirationDate).isAfter(currentDate);
```